### PR TITLE
[CI:DOCS] Fix up example description of podman-system commands

### PR DESCRIPTION
**Pull request by rhatdan**
#### Does this PR introduce a user-facing change?

```release-note
None
```

### CODE REVIEW

**Review by vrothberg — COMMENTED** [review]
LGTM

File: docs/source/markdown/podman-system-connection-add.1.md [hunk 1]
```diff
@@ -36,13 +36,24 @@ Port for ssh destination. The default value is `22`.
...
+```
 $ podman system connection add --identity ~/.ssh/dev_rsa production ssh://root@server.example.com:2222
+```
 
+Add named system connection to local unix domain socket:
+```
 $ podman system connection add testing unix:///run/podman/podman.sock
+```
 
+Add named system connection to remote tcp socket:
```
Comment (Luap99) [inline docs/source/markdown/podman-system-connection-add.1.md:55]: the URL contains localhost yet you say remote tcp socket, this contradicts itself.
I suggest you either say `local tcp socket` or change the URL in the example to something external but I do not think we should promote external tcp access at all, it is extremely insecure.

Comment (rhatdan) [reply docs/source/markdown/podman-system-connection-add.1.md]: Fixed, PTANL

File: docs/source/markdown/podman-system-connection-add.1.md [hunk 2]
```diff
@@ -36,13 +36,24 @@ Port for ssh destination. The default value is `22`.
...
+```
 $ podman system connection add --identity ~/.ssh/dev_rsa production ssh://root@server.example.com:2222
+```
 
+Add named system connection to local unix domain socket:
+```
 $ podman system connection add testing unix:///run/podman/podman.sock
+```
 
+Add named system connection to local tcp socket:
```
Comment (TomSweeneyRedHat) [inline docs/source/markdown/podman-system-connection-add.1.md:55]:
```suggestion
Add a named system connection to local tcp socket:
```

File: docs/source/markdown/podman-system-connection-add.1.md [hunk 3]
```diff
@@ -36,13 +36,24 @@ Port for ssh destination. The default value is `22`.
...
 ```
 $ podman system connection add QA podman.example.com
+```
 
+Add system connection using ssh data:
+```
 $ podman system connection add --identity ~/.ssh/dev_rsa production ssh://root@server.example.com:2222
+```
 
+Add named system connection to local unix domain socket:
```
Comment (TomSweeneyRedHat) [inline docs/source/markdown/podman-system-connection-add.1.md:50]:
```suggestion
Add a named system connection to local Unix domain socket:
```

File: docs/source/markdown/podman-system-connection-add.1.md [hunk 4]
```diff
@@ -36,13 +36,24 @@ Port for ssh destination. The default value is `22`.
 Path to the Podman service unix domain socket on the ssh destination host
 
 ## EXAMPLE
+
+Add named system connection:
 ```
 $ podman system connection add QA podman.example.com
+```
 
+Add system connection using ssh data:
```
Comment (TomSweeneyRedHat) [inline docs/source/markdown/podman-system-connection-add.1.md:45]:
```suggestion
Add a system connection using SSH data:
```

File: docs/source/markdown/podman-system-connection-add.1.md [hunk 5]
```diff
@@ -36,13 +36,24 @@ Port for ssh destination. The default value is `22`.
 Path to the Podman service unix domain socket on the ssh destination host
 
 ## EXAMPLE
+
+Add named system connection:
```
Comment (TomSweeneyRedHat) [inline docs/source/markdown/podman-system-connection-add.1.md:40]:
```suggestion
Add a named system connection:
```

File: docs/source/markdown/podman-system-connection-default.1.md [hunk 6]
```diff
@@ -10,6 +10,8 @@ podman\-system\-connection\-default - Set named destination as default for the P
 Set named ssh destination as default destination for the Podman service.
 
 ## EXAMPLE
+
+Set specified connection as default:
```
Comment (TomSweeneyRedHat) [inline docs/source/markdown/podman-system-connection-default.1.md:14]:
```suggestion
Set the specified connection as default:
```

File: docs/source/markdown/podman-system-connection-remove.1.md [hunk 7]
```diff
@@ -16,6 +16,8 @@ Delete named ssh destination.
 Remove all connections.
 
 ## EXAMPLE
+
+Remove specified system connection:
```
Comment (TomSweeneyRedHat) [inline docs/source/markdown/podman-system-connection-remove.1.md:20]:
```suggestion
Remove the specified system connection:
```

File: docs/source/markdown/podman-system-connection-rename.1.md [hunk 8]
```diff
@@ -10,6 +10,8 @@ podman\-system\-connection\-rename - Rename the destination for Podman service
 Rename ssh destination from *old* to *new*.
 
 ## EXAMPLE
+
+Rename specified connection:
```
Comment (TomSweeneyRedHat) [inline docs/source/markdown/podman-system-connection-rename.1.md:14]:
```suggestion
Rename the specified connection:
```

**Pull request merged**